openapi-v1.yaml

```diff
@@ -2854,6 +2854,58 @@ definitions:
       pagination_metadata:
         $ref: "#/definitions/PaginationMetadata"
 
+  InvitationDetails:
+    description: Object including invitation details such as the array shared or organization
+    type: object
+    properties:
+      array:
+        description: the array shared
+        x-omitempty: true
+        properties:
+          id:
+            type: string
+            description: array id
+          namespace:
+            type: string
+            description: the namespace that the array belongs to
+          name:
+            type: string
+            description: name of the array
+          file_type:
+            $ref: "#/definitions/FileType"
+            description: array file type
+            x-omitempty: true
+            x-nullable: true
+          file_properties:
```

Comment:
why we need file_properties for the redirect?

Reply:
I don't like exposing the full properties just for this. Propose alternatives if they don't cause implementation difficulties

Reply:
What if we limit the `file_properties` to only contain the `is_dashboard` indicator?